composer.json

```diff
@@ -10,14 +10,14 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": ">=5.6",
```

Comment:
So people won't be able to install on >7.0 or I am missing something?

Reply:
@norkunas, thank you for your reply.
Actually, I refer your [.travis.yml](https://github.com/norkunas/onesignal-php-api/blob/master/.travis.yml) from master branch.

It seems that the `php-5.6` test added in Travis CI build.

Reply:
Perhaps we remove the `php-5.6` from `.travis.yml` file and let the PHP version require `>=7.0` in `composer.json`?

Reply:
I would like to drop <7.0 at all :) but thats something for a new major release I think.. Ok if it works on travis than all looks good, merging then.